frontend/language/src/nb.json

```
@@ -101,7 +101,7 @@
   "app_deployment.pipeline_deployment.build_result.partiallySucceeded": "Versjonen er publisert i miljøet, men har noen feil eller mangler.",
   "app_deployment.pipeline_deployment.build_result.succeeded": "Versjonen er publisert i miljøet.",
   "app_deployment.pipeline_undeploy.build_result.failed": "Kunne ikke avpublisere appen. Prøv på nytt nå, eller vent litt og prøv igjen.",
-  "app_deployment.pipeline_undeploy.build_result.none": "Avpubliserer\u2026",
+  "app_deployment.pipeline_undeploy.build_result.none": "Avpubliserer ...",
```

Ideelt bør vi bruke ellipsesymbolet i stedet for tre punktumer.
```suggestion
  "app_deployment.pipeline_undeploy.build_result.none": "Avpubliserer …",
```

---

sant :)